crates/re_viewer/src/ui/rerun_menu.rs

```diff
@@ -164,6 +164,19 @@ impl App {
             ui.close_menu();
         }
 
+        if self
+            .re_ui
+            .checkbox(
+                ui,
+                &mut self.state.app_options.show_timestamps_in_local_timezone,
+                "Show timestamps in local timezone",
+            )
+            .on_hover_text("Convert timestamps from UTC to the local timezone")
+            .clicked()
+        {
+            ui.close_menu();
+        }
```

Comment:
I think this would be nicer as two radio buttons:

```
( ) UTC    (·) Local timezone
```

Reply:
Done!

<img width="274" alt="Screenshot 2023-09-29 at 1 10 54 PM" src="https://github.com/rerun-io/rerun/assets/1396242/50da9f8a-07e8-4af6-9f76-02772b2e8e81">